apps/site/src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,40 @@
+@use "@/lib/styles/variables.scss" as variables;
```

Comment:
Suggestion: It might be simpler to just use Bootstrap rows and columns for the icons.